<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node">
        <param name="blind" value="False" />
        <param name="img_prog_freq" value="5.0" />
        
        <param name="collect_samples" value="False" />
        <param name="min_landscape_idx" value="300" />
        <param name="max_landscape_idx" value="1000" />
        <param name="min_light_idx" value="0" />
        <param name="max_light_idx" value="70" />
        <param name="sample_period" value="5.0" />
        
        <param name="use_model" value="True" />
        <param name="model_path" value="$(find tl_detector)/logs_real/ckpt-1-310.pb" />
        <param name="perturbx" value="80" />
        <param name="perturby" value="60" />
        <param name="perturbc" value="10" />
    </node>
    <node pkg="tl_detector" type="light_publisher.py" name="light_publisher" output="screen" cwd="node"/>
</launch>
